tasks/main.yml

```diff
@@ -37,8 +37,15 @@
 
 - name: Include tasks to start service
   ansible.builtin.include_tasks: "start-service.yml"
+  when: jenkins_start_service or jenkins_users | length > 0
```

Comment:
See the comment on the next task. The service must be started in order to configure Jenkins users.

Reply:
The point of introducing `jenkins_start_service` is to support not starting Jenkins as part of running the default tasks of this role. Still starting Jenkins under some condition invalidates that assumption. 
I'm not sure why configuring users is part of this role, since it's not related to JCasC. The fact that `configure-jenkins-users.yml` is the sole user of `community.general.jenkins_script` (now that `dump-jenkins-yaml.yml` has been removed) is a strong indicator that it's "out of topic".

Reply:
I agree with @ala-ableton that overriding the user's expectation seems counter productive. Can we have an assert instead, and make sure that `jenkins_users` can't be set if `jenkins_start_service` is false?

Reply:
Yeah, I can do this, but it means that it'll complicated slightly the integration in cidre if we actually want to use this feature (since, shockingly, we are probably one of the few users of this role that want to configure Jenkins users 😅).

Reply:
See 70e32e4.